WalletWasabi.Gui/Controls/WalletExplorer/ReceiveTabViewModel.cs

```diff
@@ -153,6 +155,18 @@ public ReceiveTabViewModel(WalletViewModel walletViewModel)
 				}
 			});
 
+			SaveQRCodeCommand = ReactiveCommand.CreateFromTask(async () =>
+			{
+				try
+				{
+					await SelectedAddress?.SaveQRCodeAsync();
+				}
+				catch (Exception ex)
+				{
+					Logger.LogDebug(ex);
+				}
+			});
```

Comment:
if(SelectedAddress != null)
{
	await SelectedAddress.SaveQRCodeAsync();
}

Then subscribe to the exception of SaveQRCodeCommand and log.